benchmarks/startup.py

```diff
@@ -5,7 +5,7 @@
 
 class StartupBench(BaseBench):
     number = 50
-    repeat = (50, 100, 120.0)
+    repeat = 1
```

Comment:
it was my bad, according to [docs](https://asv.readthedocs.io/en/stable/benchmarks.html#benchmark-attributes) we run benchmark `repeat` times (with startup and teardown) , each time running `number` of times